parmed/gromacs/gromacsgro.py

```diff
@@ -253,17 +254,17 @@ def write(struct, dest, precision=3, nobox=False):
             a, b, c = reduce_box_vectors(*box_lengths_and_angles_to_vectors(
                             *struct.box))
             if all([abs(x-90) < TINY for x in struct.box[3:]]):
-                dest.write('%10.5f'*3 % (a[0]/10, b[1]/10, c[2]/10))
+                dest.write(boxfmt*3 % (a[0]/10.0, b[1]/10.0, c[2]/10.0))
```

Comment:
I had a little more time to look deeper into this.  The [format spec](http://manual.gromacs.org/current/online/gro.html) says that the unit cell dimensions are free-format, space-delimited reals.  This code actually *reduces* the default precision of unit cell vectors, which I'd prefer to avoid.  What about using this instead:

```python
boxfmt = '%%%d.%df ' % (max(varwidth, 10), max(precision, 5))
```

This way we can specify a higher precision, but will never get lower.  Also note the extra space at the end.  This is needed to strictly adhere to the whitespace-delimited policy.  There was never really a chance of going "over" as GRO files store these values in nm, but if users can specify this the chance of running into this problem goes up.

Reply:
Note that with this change, you should not have to update any of the test case tolerances.